### PR TITLE
Clean up US 1 -22

### PR DESCRIPTION
**Pull request by javier-aguilar**
- Add profile namespace to replace user namespace
- Add text to homepage
- Update profile#update logic so password is not necessary when updating user info
- Restructure navigation links
- Center login form

### CODE REVIEW

File: app/controllers/passwords_controller.rb [hunk 1]
```diff
@@ -8,13 +8,13 @@ def update
...
+      flash[:error] = "Password and confirmation must match."
       render :edit
     elsif @user.save
       redirect_to '/profile'
-      flash[:error] = "Your password has been updated"
+      flash[:success] = "Your password has been updated"
     else
-      flash[:failure] = "You are missing required fields."
+      flash[:error] = "You are missing required fields."
       render :edit
```
Comment (reid-andrew) [inline app/controllers/passwords_controller.rb:18]: Good catch here - I think there may be other places in the app (like where I've written code) we can tighten up our naming conventions on these.

File: app/controllers/profile_controller.rb [hunk 2]
```diff
@@ -17,21 +17,18 @@ def edit
...
-    elsif @user.save
-      redirect_to '/profile'
-      flash[:error] = "Your profile is updated"
+    if @user.save
+      flash[:success] = "Your profile is updated"
+      redirect_to "/profile"
     else
-      flash[:failure] = "You are missing required fields."
+      flash[:error] = @user.errors.full_messages.to_sentence
       render :edit
```
Comment (reid-andrew) [inline app/controllers/profile_controller.rb:25]: This is definitely a bit cleaner than what was there.

File: app/controllers/passwords_controller.rb [hunk 3]
```diff
@@ -8,13 +8,13 @@ def update
     @user = current_user
     @user.update(password_params)
     if password_params[:password] != password_params[:password_confirmation]
-      flash[:failure] = "Password and confirmation must match."
+      flash[:error] = "Password and confirmation must match."
```
Comment (aperezsantos) [inline app/controllers/passwords_controller.rb:11]: Like the naming here

File: config/routes.rb [hunk 4]
```diff
@@ -49,9 +48,10 @@
   patch "/profile/:id/edit", to: "profile#update"
   patch "/profile/:id/password", to: "passwords#update"
 
-  namespace :user do
+  namespace :profile do
     get "/", to: "profile#index"
```
Comment (reid-andrew) [inline config/routes.rb:52]: Good catch here - I think this will enable us to be consistent with user stories.

**Review by reid-andrew — APPROVED** [review]

File: app/views/profile/index.html.erb [hunk 5]
```diff
@@ -1,8 +1,11 @@
...
-<p>address <%=@user.street_address%></p>
-<p>city <%=@user.city%></p>
-<p>state <%=@user.state%></p>
-<p>zip <%=@user.zip%></p>
-<p>email <%=@user.email%></p>
-<%= link_to "Edit My Profile Data", "/profile/#{@user.id}/edit"%>
-<%= link_to "Edit My Password", "/profile/#{@user.id}/password"%>
\ No newline at end of file
+
+<section>
```
Comment (aperezsantos) [inline app/views/profile/index.html.erb:3]: Looks cleaner, more organized!

**Review by aperezsantos — APPROVED** [review]
It looks great, very clean and organized

**Pull request merged**